deploy/helm/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
I think if we intend only to support helm 3/provide only helm 3 install instructions, apiVersion should be bumped to v2.  ref: https://helm.sh/docs/topics/v2_v3_migration/

Reply:
I was a bit hesitant whether to do it and in the end it seemed to me that although it is clear that we do not support Helm2,  the current chart does not use any feature from `apiVersion: v2` or any other feature from Helm3. So in fact even though it is not supported, anyone who wants can still install it with Helm2. 
Bouncing to `apiVersion: v2` will force incompatibility but overall the chart is compatible. What do you think?

Reply:
makes sense to me, let's keep it v1 for now